index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <section class="container">
+      <header>
+        <img src="images/Light.png" alt="status bar" width="100%" />
+        <div class="toolbar">
+          <h1 id="logo">ROZH</h1>
+          <img id="menu-image" src="images/Icon - Menu.png" alt="Menu icon" />
+        </div>
+      </header>
+
+      <section class="headline">
+        <article class="self-info">
+          <h2>
+            I'm Rozh <br />
+            Glad to see <br />
+            you! <br />
+          </h2>
+          <p>
+            I’m a software developer! I can help you <br />
+            build a product , feature or website <br />
+            Look through some of my work and <br />
+            experience! If you like what you <br />
+            see and have a project you need coded, <br />
+            don’t hestiate to contact me. <br />
+          </p>
+          <h3>LET'S CONNECT</h3>
+        </article>
+        <div class="social-media">
+          <img src="images/Vector.png" alt="Twitter Logo" />
+          <img src="images/Linkedin icon.png" alt="Linkedin Logo" />
+          <img src="images/Vector (1).png" alt="mobile" />
+          <img src="images/Vector (2).png" alt="Github Logo" />
+          <img src="images/Vector (3).png" alt="Hand Logo" />
+        </div>
```

Comment:
- [ ] This is a list of social media links, so the list would be better wrapped with a `ul` tag with each item wrapped by the `li` tag. Since it is a list of links, each link should be wrapped with the `a` tag. Please make these changes to make your HTML file semantically correct and don't forget to add the `href` attribute for each link (if you don't have a social media of any of these platforms you can reference this page by passing the `href="#"`  to link to the top of the current page).

Reply:
Thanks so much for the feedback. I implemented the suggestion but I had to remove the div that was surrounding the my social media icons and had a class of "social-media". Instead, I gave that class name to the ul where all my li and a tags are. is that ok? @leonardodiasb